src/components/Pivot/Pivot.md

```diff
@@ -0,0 +1,30 @@
+### Notes for use
+
+Add here any notes on how to use the component, or delete this section.
+
+### Examples
+
+Basic usage:
+
+```js { "props": { "data-description": "basic" } }
+const pivotItems = [
+  { text: 'DISCOVERY', key: '0' },
+  { text: 'ALL', key: '1' },
+  { text: 'FOLLOWING', key: '2' }
+];
+
+<Pivot onChange={action('onChange')} pivotItems={pivotItems} />
+```
+
+With Dark theme:
+
+```js { "props": { "data-description": "basic" } }
```

Comment:
need to change the data-description to something like "with dark theme", this is used in the image diff filename. at the moment one of these overwrote the other so there was only one generated image diff